src/main/java/org/apache/pirk/querier/wideskies/QuerierDriverCLI.java

```diff
@@ -239,12 +242,19 @@ private boolean parseOptions()
       }
       SystemConfiguration.setProperty(CERTAINTY, getOptionValue(CERTAINTY));
 
-      if (!hasOption(QUERYNAME))
+      if (!hasOption(QUERYID))
       {
-        logger.info("Must have the option " + QUERYNAME);
+        logger.info("Must have the option " + QUERYID);
         return false;
       }
-      SystemConfiguration.setProperty(QUERYNAME, getOptionValue(QUERYNAME));
+      
+      // create a formatted date/time string
+      Date date = new Date();
+      SimpleDateFormat sdf = new SimpleDateFormat("MM.dd.yyyy_hh.mm.ss");
+      String formattedDate = sdf.format(date);
```

Comment:
I believe we decided to go with Java 8 for Pirk and the POM has been updated to reflect that.  The link u point to needs to be updated too. Pirk is officially now on Java 8. Suggest u rebase ur  code with the present master and update this PR.